### PR TITLE
Hyphenate the arguments

### PR DESCRIPTION
**Pull request by cillianderoiste**
IIUC this isn't being used anywhere, so I didn't add an alias for the older arguments. Please let me know if I should.

### CODE REVIEW

**Comment by cillianderoiste** [comment]
@jcbrand does this look OK?

**Comment by jcbrand** [comment]
@cillianderoiste Perfect, thanks!

File: src/pat-display-time.js [hunk 1]
```diff
@@ -47,8 +47,8 @@
             var date = moment(dateStr, options.format, options.locale, options.strict);
             if (options.fromNow == true) {
                 date = date.fromNow();
-            } else if (options.outputFormat.length) {
-                date = date.format(options.outputFormat);
+            } else if (options.output.format.length) {
```
Comment (adrianwhite) [inline src/pat-display-time.js:50]: Should these be hyphenated, not dot separated?

Comment (jcbrand) [inline src/pat-display-time.js:50]: No, it's correct like that. The parser creates the options object like that.

**Pull request merged**